docs/src/extend/plugins.md

```diff
@@ -62,7 +62,27 @@ export default plugin;
 module.exports = plugin;
 ```
 
-The `meta.name` property should match the npm package name for your plugin and the `meta.version` property should match the npm package version for your plugin. The easiest way to accomplish this is by reading this information from your `package.json`.
+The `meta.name` property should match the npm package name for your plugin and the `meta.version` property should match the npm package version for your plugin. The easiest way to accomplish this is by reading this information from your `package.json`, as in this example:
+
+```js
+import fs from "fs";
+
+const pkg = JSON.parse(fs.readFileSync("./package.json", "utf8"));
```

Comment:
This assumes that the current directory is the same as the directory that contains the plugin's package.json file (`"./package.json"`). That is normally not the case when the plugin is loaded from a package under `node_modules`. Probably the file name should be resolved as an absolute path or URL before being passed to `readFileSync`, because we can't make assumptions about the current directory.

```suggestion
const pkg = JSON.parse(fs.readFileSync(new URL("./package.json", import.meta.url), "utf8"));
```